lib/tether/tether.go

```diff
@@ -898,3 +898,28 @@ func killHelper(session *SessionConfig) error {
 
 	return nil
 }
+
+// Create necessary folders/files as the src/target for bind mount.
+// See https://github.com/vmware/vic/issues/489
+func createBindSrcTarget(files map[string]os.FileMode) error {
```

Comment:
suggest this goes into `tether_linux.go:Setup` - tether.go is supposed to be the pure tether logic without OS capability assumptions. Setup is the function provided for ensuring a given OS type has basic setup performed.